src/v2/guide/events.md

```diff
@@ -223,24 +223,26 @@ The `.passive` modifier is especially useful for improving performance on mobile
 
 ## Key Modifiers
 
-When listening for keyboard events, we often need to check for common key codes. Vue also allows adding key modifiers for `v-on` when listening for key events:
+When listening for keyboard events, we often need to check for specific keys. Vue allows adding key modifiers for `v-on` when listening for key events:
 
 ``` html
-<!-- only call `vm.submit()` when the `keyCode` is 13 -->
-<input v-on:keyup.13="submit">
+<!-- only call `vm.submit()` when the `key` is PageDown -->
+<input @keyup.page-down="onPageDown">
 ```
 
-Remembering all the `keyCode`s is a hassle, so Vue provides aliases for the most commonly used keys:
+In the above example, the handler will only be called if `$event.key === 'PageDown'`.
 
-``` html
-<!-- same as above -->
-<input v-on:keyup.enter="submit">
+You can directly use any valid key names exposed via [`KeyboardEvent.key`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values) as modifiers by converting them to kebab-case.
+
+### Key Codes
 
-<!-- also works for shorthand -->
-<input @keyup.enter="submit">
+The usage of `keyCode` attributes is deprecated, but can be used if needed:
```

Comment:
A better approach here IMHO would be using some `p.tip` as a warning instead. Linking "deprecated" to the MDN article would also be nice.